docs/tools.md

```diff
@@ -195,7 +195,7 @@ Flags:
   -num-partitions int
     	number of partitions in cluster (default 1)
   -partition-scheme string
-    	method used for partitioning metrics. (byOrg|bySeries|bySeriesWithTags) (default "byOrg")
+    	method used for partitioning metrics. (byOrg|bySeries|bySeriesWithTags|bySeriesWithTagsFnv) (default "byOrg")
```

Comment:
Would this be a good time to change the default to `bySeriesWithTags`?

Reply:
that requires a bit more coordination. i've created a separate ticket and ping you.